File: 1008.c
#include <stdio.h>

int main(void){
    int A,B;
    double C;
    scanf("%d %d %lf",&A,&B,&C);
    printf("NUMBER = %d\n",A);
    printf("SALARY = U$ %.2lf\n",(B*C));
}

File: Kotlin/1008.kt
fun main(args: Array<String>) {
  var number:Int = readLine()!!.toInt()
  var hour_of_workr:Int = readLine()!!.toInt()
  var hourly_value:Float = readLine()!!.toFloat()

  var salary = hour_of_workr * hourly_value

  println("NUMBER = $number")
  println("SALARY = U$ %.2f".format(salary))
}